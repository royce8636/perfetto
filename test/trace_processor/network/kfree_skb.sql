SELECT
  ts
FROM
  counter AS c
  LEFT JOIN
  counter_track AS t
  ON c.track_id = t.id
WHERE
  name GLOB "Kfree Skb"
ORDER BY ts;