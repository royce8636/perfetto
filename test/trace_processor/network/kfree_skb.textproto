packet {
  ftrace_events {
    cpu: 2
    event {
      timestamp: 10000
      pid: 200
      kfree_skb {
      }
    }
  }
}
packet {
  ftrace_events {
    cpu: 2
    event {
      timestamp: 10020
      pid: 300
      kfree_skb {
      }
    }
  }
}
packet {
  ftrace_events {
    cpu: 2
    event {
      timestamp: 20000
      pid: 200
      kfree_skb {
      }
    }
  }
}
packet {
  ftrace_events {
    cpu: 2
    event {
      timestamp: 20020
      pid: 300
      kfree_skb {
      }
    }
  }
}