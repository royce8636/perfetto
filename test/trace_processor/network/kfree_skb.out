"ts"
10000
10020
20000
20020
